server/services/reporting.js

```diff
@@ -383,53 +422,54 @@ const getRosParticipantsReport = async (region = DEFAULT_REGION_NAME) => {
         },
       },
     })
-    .find(searchOptions);
+    .find(
+      {
+        'siteJoin.body.healthAuthority': region,
+        status: 'assigned-same-site',
+      },
+      {
+        order: [
+          {
+            field: 'participant_id',
+          },
+        ],
+      }
+    );
 
   // HAs need only see the participants in their health region + participants who changed their health region and now assigned to a site withing HAs view
-  if (region !== DEFAULT_REGION_NAME) {
-    // select participants outside HAs region for changed sites
-    const additionalEntries = await dbClient.db[collections.ROS_STATUS]
-      .join({
-        participantJoin: {
-          type: 'LEFT OUTER',
-          relation: collections.PARTICIPANTS,
-          on: {
-            id: 'participant_id',
-          },
+  // select participants outside HAs region for changed sites
+  const editedEntries = await dbClient.db[collections.ROS_STATUS]
```

Comment:
So `sameSiteRosEntries` should contain all of the participants that we want, but might be missing some ROS entries for some of those participants if their site has been changed?

Then the intent here is to query the rest of the ROS statuses for each of those participants?